content/services/csv/powerdev_open_source_projects.csv

```diff
@@ -71,6 +71,7 @@
 "OpenLibm","PowerPC support for OpenLibm, a high quality, portable, standalone C mathematical library"
 "`OpenQuake`_","An Open Source Engine for seismic hazard and risk assessment uses the infrastructure to try porting to POWER architecture and see how a different architecture affects the numerical tests (the code is covered by numerical, functional and qa tests)"
 "OpenWhisk","Supports building ppc64le Docker images for OpenWhisk, a programming service that provides event based execution of functions"
+"OpenWrt","Linux operating system targeting embedded devices"
```

Comment:
This is only on the x86 cluster and should not be added here.

Reply:
I always get these wrong :(